buildspec.yml

```diff
@@ -19,7 +19,7 @@ phases:
         else
             LATEST_TAG=$(curl -i "Accept: application/vnd.github+json" -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/Privado-Inc/privado/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
             VERSION_INITIAL=${LATEST_TAG:1}
-            VERSION=$VERSION_INITIAL.$CODEBUILD_BUILD_NUMBER
+            VERSION=$VERSION_INITIAL-$CODEBUILD_BUILD_NUMBER
```

Comment:
Build numbers are supposed to be prefixed with `+`.
`-` indicates prerelease label.

Ref: https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions

Reply:
@ojaswa1942 we also tag the docker image with the given version and it doesn't allow to have `+` in tag name. So for now I am going ahead and approving this change, we may have to revisit this again.